src/Symfony/Component/ObjectMapper/Transform/MapCollection.php

```diff
@@ -21,10 +22,10 @@
  *
  * @implements TransformCallableInterface<object, T>
  */
-class MapCollection implements TransformCallableInterface
+class MapCollection implements TransformCallableInterface, ObjectMapperAwareInterface
 {
     public function __construct(
-        private ObjectMapperInterface $objectMapper = new ObjectMapper(),
+        private ?ObjectMapperInterface $objectMapper = null,
```

Comment:
@stof I'd like to introduce this change (which btw is testing the interface on callables) but I'm not sure if I should handle a BC (this was first released in the beta versio a few days ago).